src/text.ml

```diff
@@ -108,7 +108,7 @@ type action =
 
 let pp_action fmt = function
   | Invoke (mod_name, name, c) ->
-    pp fmt "(invoke %a %s %a)" pp_id_opt mod_name name pp_consts c
+    pp fmt {|(invoke%a "%s" %a)|} pp_id_opt mod_name name pp_consts c
```

Comment:
You added some `"` so the code was broken before, right ?

Reply:
yes